src/index.html

```diff
@@ -1,13 +1,413 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link
+      type="image/x-icon"
+      href="images/favicon.svg"
+      rel="icon"
+    >
+    <title>MyBiKE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-action">
+            <a href="/" class="logo">
```

Comment:
Checklist 5th and 6th items
![image](https://user-images.githubusercontent.com/37161315/170648210-c8f92f86-fdbd-4b83-a669-90a720286c9e.png)